Core/Newsletter2Go/Components/Api/Resource/NewsletterCustomer.php

```diff
@@ -337,15 +337,26 @@ public function getStreamList(
         $selectFields = array();
         $arrangedFields = $this->arrangeFields($fields);
 
-        $builder = $this->getRepositoryCustomer()
-                        ->createQueryBuilder('customer')
-                        ->innerJoin(
-                            'Shopware\Models\CustomerStream\Mapping',
-                            'mapping',
-                            Expr\Join::INNER_JOIN,
-                            'mapping.customerId = customer.id'
-                        )
-                        ->where('customer.active = true');
+
+        if($this->compareShopwareVersion('5.6.0')){
+            $builder = $this->getManager()->createQueryBuilder();
+            $builder->select(array('customer'))
+                    ->from('Shopware\Models\Customer\Customer', 'customer')
+                    ->innerJoin('Shopware\Models\CustomerStream\Mapping', 'mapping')
```

Comment:
I am not sure if this will work as wished. I think it should be like this instead:
```php
$builder->select(['customer'])
  ->from('Shopware\Models\Customer\Customer', 'customer')
  ->innerJoin('Shopware\Models\CustomerStream\Mapping', 'mapping', Expr\Join::WITH, 'mapping.customerId = customer.id')
  ->where('customer.active = true');
```
check doctrine documentation : https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/query-builder.html#line-number-5775ab09aefa638b14c776a733b4b61d2e324b74-35

This can be also useful: https://stackoverflow.com/a/15380197

Reply:
I tried it the same way as you suggest. 

In shopware there was no error anymore but there was no recipient result. 

With my solution it works in both versions and we get the same amount of recipients. 

I could show you if you want :)